dist/build.gradle

```diff
@@ -0,0 +1,141 @@
+
+apply plugin: 'es.hadoop.build'
+
+description = "Elasticsearch for Apache Hadoop"
+project.archivesBaseName = 'elasticsearch-hadoop'
+
+configurations {
+    embedded {
+        canBeResolved = true
+        canBeConsumed = false
+        transitive = false
+    }
+    dist {
+        canBeResolved = true
+        canBeConsumed = false
+        attributes {
+            attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, 'packaging'))
+        }
+    }
+}
+
+dependencies {
+    // This is only going to pull in each project's regular jar
+    embedded(project(":elasticsearch-hadoop-mr"))
+    embedded(project(":elasticsearch-hadoop-hive"))
+    embedded(project(":elasticsearch-hadoop-pig"))
+    embedded(project(":elasticsearch-spark-20"))
+    embedded(project(":elasticsearch-storm"))
+
+    // To squash Javadoc warnings
+    compileOnly(project(":elasticsearch-hadoop-mr"))
+    compileOnly(project(":elasticsearch-hadoop-hive"))
+    compileOnly(project(":elasticsearch-hadoop-pig"))
+    compileOnly(project(":elasticsearch-spark-20"))
+    compileOnly(project(":elasticsearch-storm"))
+    compileOnly("org.apache.spark:spark-catalyst_${project.ext.scala211MajorVersion}:$spark20Version")
+
+    // For Uber pom (and Javadoc to a lesser extent)
+    // TODO: Would be way better if we could pull these directly from those projects. Should we just grab the configurations?
```

Comment:
I'm pulling each dependency explicitly in order to get the POM file to generate the correct dependencies. Using regular project dependencies just makes the POM file reference the artifacts from those projects instead of the dependencies from those projects.

Reply:
If you want to bring in the dependencies too, then teh configuration you consume needs to extend the correct configuration. The `default` configuration already does this by extending `runtime`.

Reply:
After thinking about this for a little bit, having these duplicated here isn't too bad, especially since many of the dependencies might change scopes on the uberpom compared to each integration's pom file.